docs/tutorials/sample_expval_program/qiskit_runtime_expval_program.ipynb

```diff
@@ -63,7 +63,7 @@
     "- `circuits`: A single QuantumCircuit or list of QuantumCircuits to be executed on the target backend.\n",
     "\n",
     "\n",
-    "- `expectation_operators`: The operators we want to evaluate.  These can be strings of diagonal Pauli's, eg, `ZIZZ`, or custom operators defined by dictionaries. For example, the projection operator on the all ones state of 4 qubits is `{'1111': 1}`.\n",
```

Comment:
I think this is actually correct, "all ones state" is one entity so the "the" is necessary